flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/mailbox/MailboxProcessor.java

```diff
@@ -84,16 +83,30 @@
 	private final StreamTaskActionExecutor actionExecutor;
 
 	public MailboxProcessor(MailboxDefaultAction mailboxDefaultAction) {
-		this(mailboxDefaultAction, new TaskMailboxImpl(Thread.currentThread()), StreamTaskActionExecutor.IMMEDIATE);
+		this(mailboxDefaultAction, StreamTaskActionExecutor.IMMEDIATE);
 	}
 
-	public MailboxProcessor(MailboxDefaultAction mailboxDefaultAction, TaskMailbox mailbox, StreamTaskActionExecutor actionExecutor) {
+	public MailboxProcessor(MailboxDefaultAction mailboxDefaultAction,
```

Comment:
nit: those constructors have a [formatting issue ](**url**) 
- Break the argument list or chain of calls if the line exceeds limit or earlier if you believe that the breaking would improve the code readability
- If you break the line then each argument/call should have a separate line, including the first one
- Each new line should have one extra indentation (or two for a function declaration) relative to the line of the parent function name or the called entity

Reply:
fixed, thanks